src/main/java/com/stuypulse/robot/subsystems/Edwin.java

```diff
@@ -0,0 +1,216 @@
+package com.stuypulse.robot.subsystems;
+
+import static com.stuypulse.robot.constants.Ports.Edwin.*;
+import static com.stuypulse.robot.constants.Settings.Edwin.*;
+import static com.stuypulse.robot.constants.Motors.Edwin.*;
+
+import com.kauailabs.navx.frc.AHRS;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.stuypulse.stuylib.control.Controller;
+import com.stuypulse.stuylib.control.feedback.PIDController;
+import com.stuypulse.stuylib.control.feedforward.Feedforward;
+import com.stuypulse.stuylib.math.Angle;
+import com.stuypulse.stuylib.network.SmartNumber;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.DifferentialDriveKinematics;
+import edu.wpi.first.math.kinematics.DifferentialDriveOdometry;
+import edu.wpi.first.math.trajectory.TrajectoryConfig;
+import edu.wpi.first.wpilibj.SPI;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+import edu.wpi.first.wpilibj.smartdashboard.Field2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class Edwin extends Robot {
+    
+    private final CANSparkMax[] left;
+    private final CANSparkMax[] right;
+    
+    private final DifferentialDrive drivetrain;
+
+    private final RelativeEncoder leftEncoder;
+    private final RelativeEncoder rightEncoder;
+
+    private final Controller leftController, rightController;
+    private SmartNumber leftTargetSpeed, rightTargetSpeed;
+
+    private final DifferentialDriveOdometry odometry;
+    private final DifferentialDriveKinematics kinematics;
+    private final AHRS navx;
+
+    private final Field2d field;
+
+    public Edwin() {
+        left = new CANSparkMax[] {
+            new CANSparkMax(LEFT_TOP, MotorType.kBrushless), 
+            new CANSparkMax(LEFT_BOTTOM, MotorType.kBrushless)
+        };
+
+        right = new CANSparkMax[] {
+            new CANSparkMax(RIGHT_TOP, MotorType.kBrushless), 
+            new CANSparkMax(RIGHT_BOTTOM, MotorType.kBrushless)
+        };
+
+        LEFT_TOP_MOTOR.configure(left[0]);
+        LEFT_BOTTOM_MOTOR.configure(left[1]);
+
```

Comment:
Have a configure method that loops thru left and right and configures them appropiately. also set the position conversion factor AND the velocity conversion factor in that method